dts/bindings/sensor/nordic,nrf-qdec.yaml

```diff
@@ -44,3 +44,22 @@ properties:
     type: int
     description: Number of steps on the rotating wheel
     required: true
+
+  sampling-rate:
+    type: int
+    default: 4
```

Comment:
Please add an explanation behind the default value (is it power on reset value?)

Reply:
Please also add unit for this. someone may think sampling rate is in HZ, but the enums referenced below imply an interval??

Reply:
I adjusted it.